data/hai/unfettered 2 war preparations.txt

```diff
@@ -0,0 +1,2164 @@
+	substitutions
+		<brosis> "brothers and sisters"
+		<brosis> "[brothers, sisters]"
+			not "unfettered: Mera line"
+		<brethren> "brethren"
+		<brethren> "[brothers, sisters]"
+			not "unfettered: Mera line"
+		<kin> "kin"
+		<kin> "[brothers, sisters]"
+			not "unfettered: Mera line"
+		<reckless> "reckless"
+		<reckless> "[impatient, reckless]"
+			not "unfettered: Mera line"
+		<selling> "selling"
+		<selling> "[providing, selling]"
+			not "unfettered: Mera line"
+		<welcomed> "welcomed"
+		<welcomed> "[rewarded, welcomed]"
+			not "unfettered: Mera line"
+	to offer
+		has "salary: Unfettered"
+	on offer
+		conversation
+			`Not often do you see so much activity in Unfettered space. It looks like a major undertaking is about to happen, and you will probably be a part of it. When you land, Mera is already in the process of spreading a message for all Unfettered to hear, her voice somehow present across the entire spaceport.`
+			`	She speaks in Hai but she made sure to provide you with the translation on the device she gave you back when you were preparing to battle the Auxiliary. You would easily be able to read it to get a full translation.`
+				to display
+					not "unfettered: Mera line"
+			branch speak
+				not "unfettered: Merora translate"
+			`	Thankfully you spot Merora, the young Hai that helped you translate some time ago, back in the temple on this very same planet, when the elder's meeting with the public was happening.`
+				to display
+					has "unfettered: Merora translate"
+			`	As always you have your translator with you, ready for action.`
+				to display
+					has "language: Hai"
+			`	You also spot a translator or two near the stage, apparently for public use. They may be there for the occasional pirate the Unfettered may make use of, but there does not seem to be anyone using them here.`
+				to display
+					not "language: Hai"
+			choice
+				`	(Go speak with Merora.)`
+					to display
+						has "unfettered: Merora translate"
+				`	(Grab the translator.)`
+					to display
+						not "language: Hai"
+					goto speaktake
+				`	(Use your translator.)`
+					to display
+						has "language: Hai"
+					goto speak
+			`	You notice she seems to have made a few friends since the last time you saw her, and she is so busy discussing with them that she hardly notices you until you are right in front of her. She stands there for a few seconds, maw open, before gaining back focus and smiling with joy. "<first>! It's been a while." She presents her comrades to you, "This is Rick, a human that grew up in Hai space and then proved himself through other means than battles." You shake hands and present yourself. "And this is Kenep, she left Hai space some time ago, to get inspiration for her art. They work on the same ship I do. To be frank we don't see battle often, but I've had my fair share of it already, so I'm more than happy with that!" She says the last part with a bit of shame, but honesty.`
```

Comment:
```suggestion
			`	You notice she seems to have made a few friends since the last time you saw her, and she is so busy talking with them that she hardly notices you until you are right in front of her. She stands there for a few seconds, mouth open, before gaining back focus and smiling with joy. "<first>! It's been a while." She presents her comrades to you, "This is Rick, a human that grew up in Hai space and then proved himself through other means than battles." You shake hands and present yourself. "And this is Kenep. She left Hai space some time ago, to get inspiration for her art. They work on the same ship I do. To be frank we don't see battle often, but I've had my fair share of it already, so I'm more than happy with that!" She says the last part with a bit of shame, but honesty.`
```